CIP-????/README.md

```diff
@@ -0,0 +1,220 @@
+Documentation about the existing events can be found
+[here](https://github.com/input-output-hk/cardano-ledger/blob/master/docs/LedgerEvents.md).
+
+### Soft forks and Hard forks
+
+Since most ledger CIPs will involve backwards incompatible changes,
+the following two definitions are helpful:
+
+**Hard fork** - A *hard fork* is a change to the protocol (not restricted to the ledger)
+resulting in a single new block definition becoming valid.
+
+Alternatively, a hard fork is a backwards incompatible change for both
+block producers and block validators.
+
+**Soft fork** - A *soft fork* is a change to the protocol (not restricted to the ledger)
+resulting in fewer blocks being valid.
+
+Alternatively, a soft fork is a backwards incompatible change for
+block producers, but a backwards compatible change for block validators.
+
+### Serialization
+
+Transactions and blocks are serialized with
+[CBOR](https://www.rfc-editor.org/rfc/rfc7049)
+and specified with
+[CDDL file](https://www.rfc-editor.org/rfc/rfc8610).
+
+Serialization changes to the ledger are discussed in
+[CIP-80](https://github.com/cardano-foundation/CIPs/pull/372).
+
+Note that the serialisation format of the ledger state is unspecified and left as an
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+Additionally, there is significant overlap with the Ledger category around the ledger-script
+interface and the protocol parameters.
+CIPs which change these parts of Cardano should generally use the Plutus category and not the
+Ledger category, although the Editors may ask the Ledger reviewers to comment.
+
+## Specification
+
+### What merits a ledger CIP?
+
+The criterion for deciding if a change to the ledger merits a CIP is as follows:
+changes to the ledger require going through the CIP process whenever
+every implementation of the Cardano ledger needs to be standardized on the details.
+
+Bug fixes are an exception to this criterion, they do not merit a CIP except in the case
+that the fix is substantially complicated.
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+Serialization changes are another possible exception to the criterion.
+Many serialization changes can be handled as a part of the normal development process
+without the need for a CIP.
+Dramatic changes to the serialization, however, may benefit from the CIP process.
+
+The ledger rules MUST be standardized in order for consensus to be maintained,
+but things like the ledger events are more open to debate.
+
+Changes to the protocol parameter values do not require a CIP since they are
+a governance issue (see [CIP-1694]).
```

Comment:
I think this section would benefit from a little bit of rework to state more clearly what is a potential candidate for a `Ledger` CIP, and what isn't. I'd suggest to make two lists with bullet points. 

I would be keen on seeing a mention of the RSS here (in either of the list).

cc @JaredCorduan

Reply:
@KtorZ do you mean that the content is fine, but that it would help to use bullets to make it more visually clear?

What RSS are you referring to?

Reply:
> @JaredCorduan do you mean that the content is fine, but that it would help to use bullets to make it more visually clear?

Yes, the suggestion is purely _structural_. From the perspective of a contributor, I'd like to easily grasp in one look what are suitable topics for a ledger CIP and what aren't.

> What RSS are you referring to?

I am referring to anything that touches the rewards and incentives schemes. This is a common topic of discussion in CIPs; ranging from suggestions of new values for protocol parameters to changes in the reward equations (in particular w.r.t to the pledge influence) or complete redesign of the incentives. 

Those CIPs have historically gotten very little attention and are hard to move forward. I'd appreciate clarity as to whether the `Ledger` category now cover them, or not. If the intent is to cover this area, then I'd also love the process to be specified in more detail in this proposal. So far, it seems that RSS questions have required the involvement of IO research but the process hasn't always / ever been smooth as there's little availability from the research side. 

Plus, all CIPs in this situation never got a clear answer from IO / the ledger team regarding their "acceptance". That's why from an edition perspective, we've recently decided to simply mark all of them as _Inactive_ because we don't have any procedure by which we can propose them and move them forward. 

If this proposal still doesn't address the case of RSS proposals, then let's state it clearly here too.

Reply:
> I'd like to easily grasp in one look what are suitable topics for a ledger CIP and what aren't.

I definitely appreciate your help to make this more readable and friendly. Really there is just one rule:

```
every implementation of the Cardano ledger needs to be standardized on the details.
```

The paragraphs below this, that I think you are suggesting making into a bullet list, are just elaborations on this and exceptions. Maybe I could bold the main criterion sentence? Would adding bullets to the other things make the relevance of the main criterion seem less apparent?

Reply:
regarding the rewards and incentives schemes, my hope was that it would be clear that indeed they are in scope:

```
every implementation of the Cardano ledger needs to be standardized on the details.
```

The caveat is listed below:

```
 For example, changes to the incentives model (such as changing the reward calculation)
  should include a proper game theoretic analysis.
```

Of course, as you can see, I've given no guidance on what is "proper" or who decides. 

> Those CIPs have historically gotten very little attention and are hard to move forward

I think it is more that there are only so many things we can do at once, and we have been focused on babbage and voltaire.

Reply:
@JaredCorduan yes I think it does appear that the need to categorise RSS proposals _somehow_ would be satisfied by giving them a `Ledger` category... while the comment you quote directly above also indicates that the consideration of these proposals will generally depend on expertise & review outside the Ledger team (i.e. a "disclaimer" so that their responsibility would not fall exclusively upon them).

So in my opinion this question has no reason to block your proposal as it currently stands... but I'll leave this conversation open so we can see it in our CIP meeting (1 hour from now). (cc @KtorZ)